### ISSUE TITLE
Chrome lo

### ISSUE THREAD

**Issue by rhandberg** [johnculviner/jquery.fileDownload#142]

**Comment by AndrewFreese** [comment]
If you change line 159 to   
} else if (userAgent.indexOf('android') !== -1 && userAgent.indexOf('chrome') === -1) {

Chrome on android works fine. I guess you'd need to add a firefox clause too.

**Comment by johnculviner** [comment]
good call lemme get a fix for this!

**Comment by Greibit** [comment]
How about this?

**Comment by johnculviner** [comment]
Yeah... just had a baby so stuff is getting a little difficult to keep up with, anyone using the library right now (i am not) having this issue mind taking a stab at it and i'm happy to take a PR!
